app/Forms/UpdateProfileForm.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Forms;
+
+use Kris\LaravelFormBuilder\Form;
+use Carbon\Carbon;
+use App\Member;
+use App\College;
+use App\Course;
+use App\Scholarship;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Auth;
+use DB;
+
+class UpdateProfileForm extends Form
+{
+	private $details = null;
+	public function __construct()
+	{
+		$user = Auth::user();
+		$this->details = DB::table('members')->where('crsid', $user->crsid)->first();
+	}
+	
+    public function buildForm()
+    {
+        $this
+            ->addBio()
+            ->add('date_of_birth', 'date', ['default_value' => $this->details->date_of_birth])
```

Comment:
That's true. @tanhanliang , any insight on why this was needed? You probably have a good reason for this since it wasn't required in the original MembersSignupForm. (See my other comment regarding the constructor too)